src/include/uthenticode.h

```diff
@@ -48,6 +48,7 @@ DECLARE_ASN1_FUNCTIONS(Authenticode_SpcIndirectDataContent)
  * So we wrap it here for use with unique_ptr.
  */
 void OpenSSL_free(void *ptr);
+void SK_X509_free(stack_st_X509 *ptr);
```

Comment:
Thanks! Mind adding a small comment here, similar to the one above `OpenSSL_free` (explaining specifically that it's an OpenSSL 3+ change)?